skycoin/skywire
# Improve Windows Installer Script

Did you run `make format && make check`? Yes

Fixes #-

 Changes:	
- change/refactor `script.ps1` of windows installer
- add `win-installer-latest` to Makefile for create installer on last release. (Currently, we don't have needed archive file for v0.6.0, but will available for v1.0.0 release)
- add `CUSTOM_VERSION` parameter to `win-installer` command to create installer for specific version. Could use for both `released` or `pre-release` releases.

How to test this PR:
- use `make win-installer CUSTOM_VERSION=v1.0.0-rc5` for test this

Except Behavior:
- Console logs should be like this:
  
  ![image](https://user-images.githubusercontent.com/79150699/173054544-19d56387-8102-44b4-beaa-ae70134dbc30.png)

- And two msi installer file should created by these names:
  - `skywire-systray-v1.0.0-rc5-windows-amd64.msi`
  - `skywire-systray-v1.0.0-rc5-windows-386.msi`
